javascript/javascript1/week2/homework/main-1.js

```diff
@@ -0,0 +1,21 @@
+//FREECODECAMP
+console.log("My FreeCodeCamp username is @veronika_burduzhan");
+
+//FLIGHT BOOKING FULLNAME FUNCTION
+function getFullname(firstname, surname, useFormalName) {
+  let fullname = firstname + " " + surname;
+  if (firstname === " " || surname === " ") {
+    return "Please, fill out your name!";
+  }
+  if (useFormalName === true) {
+    return "Lord" + " " + fullname;
+  } else return fullname;
```

Comment:
Although this is totally valid javascript, I would recommend adding curly braces here like
```suggestion
  } else {
    return fullname;
  }
```
a few reasons why:
- consistency, since you used curly braces above
- if you later on decide to add more lines of code in the `else` block, and forget to add the braces, your code will break in unexpected ways
- it makes it easier to read quickly since the `if` and `else` blocks have the same indentation

Reply:
> Nice work! Seems like you have understood functions well :) I have left some comments, mostly about style.

Thanks for your comments🖤  P.S. I'll check my VSCode, maybe I didn't set Prettier as a format on save.